libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -238,6 +241,7 @@ constructor(
      * @see [registerTripSessionStateObserver]
      */
     fun stopTripSession() {
+        setRoutes(emptyList())
```

Comment:
Do we want to empty the routes list (`DirectionsSession`) when a session is stopped 🤔? Refs. https://github.com/mapbox/mapbox-navigation-android/pull/2618

Reply:
It is need to clear routes. If we don't do that, after restarting session we will have cached route. Do we have any other way to clear route?